frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesList.cy.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable camelcase */
+import { RelationshipKF, ResourceTypeKF } from '~/concepts/pipelines/kfTypes';
+import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { mockDataSciencePipelineApplicationK8sResource } from '~/__mocks__/mockDataSciencePipelinesApplicationK8sResource';
+import { mockK8sResourceList } from '~/__mocks__/mockK8sResourceList';
+import { mockNotebookK8sResource } from '~/__mocks__/mockNotebookK8sResource';
+import { buildMockPipeline, buildMockPipelines } from '~/__mocks__/mockPipelinesProxy';
+import {
+  buildMockPipelineVersion,
+  buildMockPipelineVersions,
+} from '~/__mocks__/mockPipelineVersionsProxy';
+import { mockProjectK8sResource } from '~/__mocks__/mockProjectK8sResource';
+import { mockRouteK8sResource } from '~/__mocks__/mockRouteK8sResource';
+import { mockSecretK8sResource } from '~/__mocks__/mockSecretK8sResource';
+import { mockStatus } from '~/__mocks__/mockStatus';
+import { pipelinesTable } from '~/__tests__/cypress/cypress/pages/components/PipelinesTable';
+
+const projectName = 'test-project-name';
+
+describe('PipelinesList', () => {
+  beforeEach(() => {
+    initIntercepts();
+  });
+
+  it('renders the page with pipelines table data', () => {
+    cy.visitWithLogin(`/pipelines/${projectName}`);
+    pipelinesTable.find();
+  });
+});
+
+const initIntercepts = () => {
+  cy.intercept('/api/status', mockStatus());
+  cy.intercept('/api/config', mockDashboardConfig({}));
+  cy.intercept(
+    {
+      pathname: `/api/k8s/apis/datasciencepipelinesapplications.opendatahub.io/v1alpha1/namespaces/${projectName}/datasciencepipelinesapplications/pipelines-definition`,
+    },
+    mockDataSciencePipelineApplicationK8sResource({ namespace: projectName }),
+  );
+  cy.intercept(
+    {
+      pathname: `/api/k8s/apis/kubeflow.org/v1/namespaces/${projectName}/notebooks`,
+    },
+    mockK8sResourceList([mockNotebookK8sResource({ namespace: projectName })]),
+  );
```

Comment:
Are notebooks needed to be mocked here?

Reply:
I don't know the ins and outs of the KF API calls, but without this, the `secrets` call on the page fails for the project name selected.

Reply:
Yeah, sometimes folks want to see all things indicating success even with mocks, but I'm onboard with only mocking if necessary for the test(s). 

Also, once you sent that comment last week I started using that "mock" command instead locally, so the most recent changes were tested/written while using that command instead.